examples/amm/contract.py

```diff
@@ -68,8 +64,8 @@ def bootstrap(self, seed: PaymentTransaction, a_asset: Asset, b_asset: Asset) ->
         """
         assert not self.pool_token, "application has already been bootstrapped"
         self._check_is_governor()
-        assert Global.group_size() == 2, "group size not 2"
-        assert seed.receiver == Global.current_application_address(), "receiver not app address"
+        assert op.Global.group_size() == 2, "group size not 2"
```

Comment:
I feel like `Global` and `Transaction` shouldn't be in `op`, but should be standalone?

Reply:
Or alternatively, maybe they could be lit up into `self` via the base `Contract` class as convenient aliases?

Reply:
Yeah we plan to make some more changes to make here, for Global and Transaction, any values that don't change or take arguments will be converted to properties.

I think Global and Transaction should remain in `puyapy.op` as they represent a 1:1 mapping to the underlying teal ops. But it would also think it is reasonable to also include them in the `puyapy` module as aliases, since they are commonly used.

Having them available via `self` is tempting from a discoverability point of view, but the main issue is they are contextual info of the current transaction rather than contract state. Currently all other things on `self` relate directly to state values, but keen to hear thoughts on this.